lib/add_bc_user.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+echo "This script add's a user with password to the bridghead"
+read -p 'Username: ' bc_user
+read -sp 'Password: ' bc_password
+
+echo 
+
+bc=$(docker run --rm -ti xmartlabs/htpasswd $bc_user $bc_password)
```

Comment:
Please use https://hub.docker.com/_/httpd?tab=tags instead of xmartlabs/htpasswd

Reply:
Fixed